### ISSUE TITLE
Permanent Harvestron Timer

### ISSUE THREAD

**Issue by qldhighway** [Alorel/avabur-improved#2]
The same functionality as the Construction Timer, just for the harvestron.

**Comment by Alorel** [comment]
Yep, it's on the radar - will be done when I eventually start working on the script again
